operations/companycontracts.md

```diff
@@ -83,3 +94,254 @@ Returns all contracts between the enterprise and other companies.
+        "ContactPerson": "Sample person",
+        "ContactEmail": "sample-person@email.com",
+        "AdditionalContactInfo": null,
+        "Notes": null
+    }
+}
+```
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `TravelAgencyContracts` | array of [Travel agency contract parameters](#travel-agency-contract-parameters) | required | Information about travel agency contracts to be created. |
+
+#### Travel agency contract parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ServiceId` | string | required | Unique identifier of the [Service](services.md#service) the travel agency contract is created to. |
+| `CompanyId` | string | required | Unique identifier of the [Company](companies.md#company) the travel agency contract is issued to. |
+| `CommissionIncluded` | boolean | optional | Whether commission of the travel agency is included in the rate. |
+| `Commission` | number | optional | Commission of the travel agency. |
+| `ChannelManagerAbsoluteAdjustment` | number | optional | Flat fee added to (or subtracted from) the reservation price when coming from Channel Managers. |
+| `ChannelManagerRelativeAdjustment` | number | optional | Percentage of the reservation price added to (or subtracted from) price when coming from Channel Managers. |
+| `Options` | [Travel agency contract options](#travel-agency-contract-options) | required | Options of the travel agency contract. |
+| `AccountingCode` | string | optional | Accounting code of the travel agency contract. |
+| `InvoiceDueInterval` | string | optional | The maximum time, when the invoice has to be be paid in ISO 8601 duration format. |
+| `ChannelManagerBusinessSegmentId` | string | optional | Unique identifier of the [Business segment](businesssegments.md#business-segment) used for incoming reservations originating from Channel Managers, for this particular contract. |
+| `ContactPerson` | string | optional | Contact person of the travel agency. |
+| `ContactEmail` | string | optional | Contact email of the travel agency. |
+| `AdditionalContactInfo` | string | optional | Additional contact info of the travel agency. |
+| `Notes` | string | optional | Additional notes of the travel agency contract. |
+
+### Response
+
+```javascript
+{
+    "TravelAgencyContracts": [
+        {
+            "Id": "0078f370-3787-43dc-a615-af150066bb88",
+            "ServiceId": "c8f88563-dc60-47f3-aca3-af150065d951",
+            "CompanyId": "bfd5667b-533f-424f-860d-af150065f4d6",
+            "IsActive": true,
+            "CommissionIncluded": true,
+            "Commission": 0.1,
+            "ChannelManagerAbsoluteAdjustment": 0.1,
+            "ChannelManagerRelativeAdjustment": 0.1,
```

Comment:
NIT: The response should correlate with the request, also `CompanyId` is different.